Sources/OktaOidc/Common/Internal/Tasks/OktaOidcTask.swift

```diff
@@ -39,10 +39,7 @@ class OktaOidcTask {
 
             callback(OKTServiceConfiguration(discoveryDocument: oidConfig), nil)
         }, onError: { error in
-            let responseError =
-                "Error returning discovery document: \(error.localizedDescription). Please" +
-                " check your PList configuration"
-            callback(nil, OktaOidcError.APIError(responseError))
+            callback(nil, error)
```

Comment:
No need to return `OktaOidcError.APIError` because `error` argument is `OktaOidcError` type.